### ISSUE TITLE
Tumblr RSS-feeds have GDPR screen

### ISSUE THREAD

**Issue by jawsper** [jawsper/feedreader#14]
Instead of fetching the RSS-feed, tumblr supplies the feedparser with a GDPR screen. 

Possible solution at https://discourse.tt-rss.org/t/change-on-tumblr-rss-feeds-not-working/1158/26

https://sentry.io/feedreader/feedreader/issues/681939475/

```
DoesNotExist: Feed matching query does not exist.
  File "feedreader/functions/feeds.py", line 25, in add_feed
    feed = Feed.objects.get( xmlUrl = feedXmlUrl )
  File "django/db/models/manager.py", line 82, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "django/db/models/query.py", line 403, in get
    self.model._meta.object_name

KeyError: 'title'
  File "site-packages/feedparser.py", line 398, in __getattr__
    return self.__getitem__(key)
  File "site-packages/feedparser.py", line 356, in __getitem__
    return dict.__getitem__(self, key)

AttributeError: object has no attribute 'title'
(6 additional frame(s) were not displayed)
...
  File "feedreader_api/functions.py", line 11, in post
    return JsonResponse(self.get_response(request.user, request.POST))
  File "feedreader_api/api0/views/outline.py", line 22, in get_response
    result = add_feed(user, args['url'])
  File "feedreader/functions/feeds.py", line 27, in add_feed
    feed = load_feed( feedXmlUrl )
  File "feedreader/functions/feeds.py", line 16, in load_feed
    if data.feed.title:

AttributeError: object has no attribute 'title'
```

**Comment by jawsper** [comment]
Sentry issue: [FEEDREADER-1D](https://sentry.io/organizations/feedreader/issues/1226805507/?referrer=github_plugin)

**Comment by jawsper** [comment]
Fixed in 8d0e67623fd27edfbd9b8289a94cbba82086da32
